src/components/Checkbox/Checkbox.st.css

```diff
@@ -0,0 +1,101 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Checkbox;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultTextColor: color-5;
+    DefaultBorderColor: color-5;
+    DefaultIconColor: color-8;
+    DisabledColor: color-3;
+    Font: font("{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}");
+}
+
+:vars {
+    /*Overrides*/
+    TextColor: --overridable;
+    BorderColor: --overridable;
+    IconColor: color-8;
+}
+
+:vars {
+    CurrentBorderColor: color(fallback(value(BorderColor), value(DefaultBorderColor)));
+    CurrentIconColor: color(fallback(value(IconColor), value(DefaultIconColor)));
+    CurrentTextColor: color(fallback(value(TextColor), value(DefaultTextColor)));
+}
+
+.root {
+    -st-extends: Checkbox;
+    -st-states: checked, disabled, error;
+
+    display: inline-flex;
+    align-items: center;
+}
+
+.icon {
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+
+    border: 1px solid;
+    border-color: applyOpacity(value(CurrentBorderColor), 0.6);
+
```

Comment:
The only comment I got from the designers, is that the box is missing a `border-radius` of 1 px

Reply:
Added